cli/client.go

```diff
@@ -1412,6 +1457,61 @@ func (c *viamClient) machinesPartCopyFilesAction(
 	return nil
 }
 
+func (c *viamClient) machinesPartGetFTDCAction(
+	ctx *cli.Context,
+	flagArgs machinesPartGetFTDCArgs,
+	logger logging.Logger,
+) error {
+	args := ctx.Args().Slice()
+	var targetPath string
+	switch numArgs := len(args); numArgs {
+	case 0:
+		var err error
+		targetPath, err = os.Getwd()
+		if err != nil {
+			return err
+		}
+	case 1:
+		targetPath = args[0]
+	default:
+		return wrongNumArgsError{numArgs, 0, 1}
+	}
+
+	globalArgs, err := getGlobalArgs(ctx)
```

Comment:
(minor) given that we're already creating global args in the `MachinesPartGetFTDCAction` above, can we just pass those into this function and avoiding creating again?